### DISCUSSION TITLE
How would I add Calculation Items to the BPA TODO rule?

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#443] [🙋♀️TE3 Q&A] [answered]
Is there a way to adjust the current rule or add a new rule where TODO appears in the expression of a calc item it will show up in the BPA?

**Comment by otykier** [comment, marked as answer]
Hi @edhans 
This should be possible simply by including "Calculation Items" in the **Applies to** dropdown:

![image](https://user-images.githubusercontent.com/8976200/155949511-29e9e919-9ac5-42f7-b464-8fa844a54e7e.png)

**Reply by edhans** [reply]
Well, that was pretty simple. I was looking for some other name of an object. 🙄
